runtime-attach/README.md

```diff
@@ -10,10 +10,27 @@ The attachment will _not_ be initiated in the following cases:
 * The attachment is not requested from the _main_ thread
 * The attachment is not requested from the `public static void main(String[] args)` method
 * The agent is already attached
-* The application is running on a JRE (a JDK is necessary)
-* The application is running on a read-only file system
+* The application is running on a JRE (a JDK is necessary, see [this issue](https://github.com/raphw/byte-buddy/issues/374))
+* The temporary directory should be writable
+
+To use runtime attachment, you have to add the following dependency:
+
+_Maven_
+```xml
+<dependency>
+  <groupId>io.opentelemetry.contrib</groupId>
+  <artifactId>opentelemetry-runtime-attach</artifactId>
+  <version>...</version>
```

Comment:
don't we also need the agent itself in our dependencies? Or does runtime-attach already depend on the agent transitively?

Reply:
This dependency embeds the OpenTelemetry agent JAR. I have added a sentence to the readme.